src/main/java/picard/vcf/GenotypeConcordanceStates.java

```diff
@@ -45,7 +47,7 @@
         public static TruthState getHom(final int alleleIdx) {
             if (alleleIdx == 0) return HOM_REF;
             if (alleleIdx == 1) return HOM_VAR1;
-            assert false;
+            ValidationUtils.validateArg(false,"Shouldn't be here.");
```

Comment:
In this case, why not `throw new PicardException("Invalid allele index: " + alleleIdx);`? This doesn't seem like a proper use of `validateArg()`.

Reply:
@yfarjoun Wait, I lied, you didn't respond to Phil here.